devtools/cli/src/main/java/io/quarkus/cli/build/MavenRunner.java

```diff
@@ -142,16 +147,21 @@ public Integer projectInfo(boolean perModule) throws Exception {
     }
 
     @Override
-    public Integer updateProject(String targetPlatformVersion, String targetPlatformStreamId, boolean perModule)
+    public Integer updateProject(String targetPlatformVersion, String targetPlatformStream, boolean perModule)
             throws Exception {
         ArrayDeque<String> args = new ArrayDeque<>();
         setMavenProperties(args, true);
-        args.add("quarkus:update");
+        final ExtensionCatalog extensionCatalog = ToolsUtils.resolvePlatformDescriptorDirectly(
+                ToolsConstants.QUARKUS_CORE_GROUP_ID, null,
+                Version.clientVersion(),
+                QuarkusProjectHelper.artifactResolver(), MessageWriter.info());
```

Comment:
We'll want to do that earlier and not based on the cli version. I can help with that in a follow up PR.

Reply:
@aloubyansky  This is just for the quarkus maven plugin key resolution, we said we would use the CLI version for this right?

Reply:
The CLI version is not guaranteed to be recent enough. We'll need the target platform version.